### PR TITLE
VIH-10353 - Admin web Unable to sent notifications for Edit hearings

### PR DESCRIPTION
**Pull request by marcogagliardi**
[VIH-10353](https://tools.hmcts.net/jira/browse/VIH-10353)

### CODE REVIEW

**Review by oliver-scott — APPROVED** [review]
looks good. We should ideally have tests for the mapper to protect against regression issues. Probably one to tackle through separate tech debt

File: NotificationApi/NotificationApi.UnitTests/Mappers/NotificationMappers/MapToHearingAmendmentNotificationTests.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+using System.Collections.Generic;
+using FluentAssertions;
+using NotificationApi.Common;
+using NotificationApi.Contract;
+using NotificationApi.Contract.Requests;
+using NotificationApi.Extensions;
+using NotificationApi.Services;
+using NUnit.Framework;
+
+namespace BookingQueueSubscriber.UnitTests.Mappers.NotificationMappers
```
Comment (oliver-scott) [inline NotificationApi/NotificationApi.UnitTests/Mappers/NotificationMappers/MapToHearingAmendmentNotificationTests.cs:11]: Perhaps name this and the folder `NotificationParameterMapper`, for consistency with the class name

File: NotificationApi/NotificationApi.UnitTests/Mappers/NotificationMappers/MapToHearingAmendmentNotificationTests.cs [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+{
+    public class MapToHearingAmendmentNotificationTests
+    {
+
+        [Test]
+        public void should_map_to_amendment_notification()
+        {
+            //Arrange
+            const NotificationType expectedNotificationType = NotificationType.HearingAmendmentEJudJoh;
```
Comment (oliver-scott) [inline NotificationApi/NotificationApi.UnitTests/Mappers/NotificationMappers/MapToHearingAmendmentNotificationTests.cs:20]: this line isn't used - can be removed

**Review by oliver-scott — COMMENTED** [review]
nice. Just added some suggestions

File: NotificationApi/NotificationApi.UnitTests/Mappers/NotificationMappers/MapToHearingAmendmentNotificationTests.cs (hunk 2 above)
Comment (marcogagliardi) [reply NotificationApi/NotificationApi.UnitTests/Mappers/NotificationMappers/MapToHearingAmendmentNotificationTests.cs:20]: done

File: NotificationApi/NotificationApi.UnitTests/Mappers/NotificationMappers/MapToHearingAmendmentNotificationTests.cs [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+using NotificationApi.Services;
+using NUnit.Framework;
+
+namespace BookingQueueSubscriber.UnitTests.Mappers.NotificationMappers
+{
+    public class MapToHearingAmendmentNotificationTests
+    {
+
+        [Test]
+        public void should_map_to_amendment_notification()
```
[the remark this answers is not in the document]
Comment (marcogagliardi) [reply NotificationApi/NotificationApi.UnitTests/Mappers/NotificationMappers/MapToHearingAmendmentNotificationTests.cs:17]: done

File: NotificationApi/NotificationApi.UnitTests/Mappers/NotificationMappers/MapToHearingAmendmentNotificationTests.cs (hunk 1 above)
Comment (oliver-scott) [reply NotificationApi/NotificationApi.UnitTests/Mappers/NotificationMappers/MapToHearingAmendmentNotificationTests.cs]: could we make it singular - `NotificationParameterMapper`

(both the folder and namespace)

**Review by oliver-scott — APPROVED** [review]

**Pull request merged**